src/main/resources/application.yaml

```diff
@@ -19,7 +19,7 @@ server:
     application-display-name: AnaLog
 logging:
   # path to default logging configuration file; can be overridden in external application.yaml 
-  config: classpath:logging/logback-console.xml
+  config: config/logback.xml
```

Comment:
I'm not sure it's a change for the better.
The idea behind this config property and the corresponding file is to provide a kind of fallback for the logging in case of running a bare application JAR file (i.e. with no configs around). Since this property is internal and it relies to the internal config file, together they will always provide a way for the application to properly configure its logging. This was meant not only for incorrect application usage (which is obviously unpredictable) but also for deployment in Docker containers where logging is usually redirected to stdout and external configuration should be set to minimum.

Reply:
I have reverted my changes